plugins/CoreVisualizations/templates/_dataTableViz_htmlTable.twig

```diff
@@ -21,6 +21,7 @@
         {% else %}
             {% set rowIndex = properties.filter_offset|default(0) + 1 %}
             {%- for rowId, row in dataTable.getRows() -%}
+            {% if rowId != constant('Piwik\\DataTable::ID_TOTALS_ROW') %}
```

Comment:
@diosmosis just double checking... in my Datatable class in your branch I'm not seeing that constant.

Reply:
I think you're looking at old code, it's not there if I look at "files changed".